pkgs/development/python-modules/mdpo/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, polib
+, pymd4c
+}:
+
+buildPythonPackage rec {
+  pname = "mdpo";
+  version = "0.3.0";
```

Comment:
At least ``pymd4c`` is python3.6 plus and fails to build on python27. Can you change that and check the other packages for similar version restrictions?

Reply:
mdpo is Python3.6+ also and the current version is 0.3.5